README.md

```diff
@@ -399,6 +414,9 @@ result = Profile::Create.call(params: {
 
 ```ruby
 class Profile::Create < Opera::Operation::Base
+  context_accessor :profile
```

Comment:
Maybe we can leave some examples in readme with context[:] and dependencies[:] syntax to saw both options?

Reply:
Yes. I left, for example this one: https://github.com/Profinda/opera#user-content-success